packages/interactivity/src/test/store.ts

```diff
@@ -0,0 +1,311 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
```

Comment:
Instead of using this line, the other disabled rules should be re-enabled. In general, disabling these rules at the top of the file should be avoided, and insted it should be applied with `disable` and `enable` pair closer to the usage.